src/api/mutant/StrykerNode.ts

```diff
@@ -0,0 +1,5 @@
+interface StrykerNode extends ESTree.Node {
+  nodeID: number;
```

Comment:
Ok, i might be nitpicking here, but i think the casing should be `nodeId` in this case. `Id` is short for `Identifier`. Abbreviations should always be written in PascalCase (in contrast to 2 letter acronyms). But i might be alone in this ;) so you can leave it if you want.